test/unit/com/unifina/service/CommunityOperatorServiceSpec.groovy

```diff
@@ -127,17 +127,4 @@ class CommunityOperatorServiceImplSpec extends Specification {
 			assert results.get(i).response.statusCode == 200
 		}
 	}
-
-	void "test socket read timeout"() {
```

Comment:
Should we keep this test? It seems useful.

If you want to manipulate the config for the purposes of the unit test (set a lower timeout, etc.), you can certainly do so, see an example [here](https://github.com/streamr-dev/engine-and-editor/blob/master/test/unit/com/unifina/service/NodeServiceSpec.groovy#L18). 

NOTE: It may require you to switch to accessing the config via `grailsApplication` instead of the static `Holders` object. In my experience, it's harder to manipulate the static config in unit tests (it may not change even if you manipulate the config in the unit test). For this reason (better testability), accessing config via `grailsApplication` is preferred.